commit ef68d9acc1df9584f183f1434a5bebc53e565dc7
Author: Hong Jen Yee (PCMan) <pcman.tw@gmail.com>
Date:   Thu May 8 23:53:49 2014 +0800

    Set library dirs properly in qtxdg-config and use files.

diff --git a/cmake/qtxdg-config.cmake.in b/cmake/qtxdg-config.cmake.in
index edb408b..e8b481d 100644
--- a/cmake/qtxdg-config.cmake.in
+++ b/cmake/qtxdg-config.cmake.in
@@ -7,6 +7,8 @@
 #
 #  QTXDG_INCLUDE_DIRS  - The QtXdg include directory
 #
+#  QTXDG_LIBRARY_DIRS  - The QtXdg library directory
+#
 #  QTXDG_LIBRARIES     - The libraries needed to use QtXdg
 #
 #  QTXDG_USE_FILE      - The variable QTXDG_USE_FILE is set which is the path
@@ -35,6 +37,8 @@ set(QTXDG_LIBRARY       @QTXDGX_LIBRARY_NAME@)
 set(QTXDG_LIBRARIES     ${QTXDG_LIBRARY})
 set(QTXDG_INCLUDE_DIRS  ${QTXDG_INCLUDE_DIR})
 
+set(QTXDG_LIBRARY_DIRS  @CMAKE_INSTALL_FULL_LIBDIR@)
+
 set(QTXDG_USE_FILE      ${CMAKE_CURRENT_LIST_DIR}/@QTXDGX_FILE_NAME@_use.cmake)
 set(QTXDG_FOUND 1)
 
diff --git a/cmake/qtxdg_use.cmake b/cmake/qtxdg_use.cmake
index a5a3210..93c93a6 100644
--- a/cmake/qtxdg_use.cmake
+++ b/cmake/qtxdg_use.cmake
@@ -8,3 +8,4 @@ include(${QT_USE_FILE})
 set(QTXDG_QT_LIBRARIES ${QT_LIBRARIES})
 
 include_directories(${QTXDG_INCLUDE_DIRS})
+link_directories(${QTXDG_LIBRARY_DIRS})
